docs/internal/maintenance/SystemTesting.md

```diff
@@ -52,28 +58,20 @@ The set of configurations must include at least:
    1. Prepare a machine
       1. Allocate a machine with the architecture specified by the configuration.
       2. Follow the preparation document for the operating system specified by the configuration.
-         * [CentOS-Preparation]
          * [Debian-Preparation]
          * [FreeBSD-Preparation]
          * [Ubuntu-Preparation]
          * Rocky-Linux-Preparation (TBD)
 
    2. Install Zonemaster LDNS
-      1. Make sure the [requirements for IDN support] are satisfied.
-      2. Make sure that OpenSSL is installed.
-      3. Install the preliminary distribution tarball for zonemaster-ldns.
+      1. Install dependencies according to the [LDNS installation] instructions.
+      2. Install the preliminary distribution tarball for zonemaster-ldns.
 
          ```sh
          sudo cpanm Zonemaster-LDNS-${LDNS_VERSION}.tar.gz
          ```
```

Comment:
It depends on the OS, if internal LDNS should be installed or not. I think it is better to write that the installation instruction should be followed, but replaces Zonemaster::LDNS with Zonemaster-LDNS-${LDNS_VERSION}.tar.gz